notebooks/02_analyses/sentiment_analysis.ipynb

```diff
@@ -0,0 +1,776 @@
+{
```

Comment:
<p>Do you want to take this print statement out of the for loop so that we only see one print out consisting of all months rather than some months being printed several times?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>